packages/trader/src/AppV2/Components/Filter/contract-type-filter.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import Chip from 'AppV2/Components/Chip';
+import { ActionSheet, Checkbox } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+
+type TContractTypeFilter = {
+    contractTypeFilter: string[] | [];
+    setContractTypeFilter: (filterValues: string[]) => void;
+};
+
+const availableContracts = [
+    { tradeType: <Localize i18n_default_text='Accumulators' />, id: 'Accumulators' },
+    { tradeType: <Localize i18n_default_text='Vanillas' />, id: 'Vanillas' },
+    { tradeType: <Localize i18n_default_text='Turbos' />, id: 'Turbos' },
+    { tradeType: <Localize i18n_default_text='Multipliers' />, id: 'Multipliers' },
+    { tradeType: <Localize i18n_default_text='Rise/Fall' />, id: 'Rise/Fall' },
+    { tradeType: <Localize i18n_default_text='Higher/Lower' />, id: 'Higher/Lower' },
+    { tradeType: <Localize i18n_default_text='Touch/No touch' />, id: 'Touch/No touch' },
+    { tradeType: <Localize i18n_default_text='Matches/Differs' />, id: 'Matches/Differs' },
+    { tradeType: <Localize i18n_default_text='Even/Odd' />, id: 'Even/Odd' },
+    { tradeType: <Localize i18n_default_text='Over/Under' />, id: 'Over/Under' },
+];
+
+const ContractTypeFilter = ({ contractTypeFilter, setContractTypeFilter }: TContractTypeFilter) => {
+    const [isDropdownOpen, setIsDropdownOpen] = React.useState(false);
+    const [changedOptions, setChangedOptions] = React.useState<string[]>(contractTypeFilter);
+
+    const onActionSheetClose = () => {
+        setIsDropdownOpen(false);
+        setChangedOptions(contractTypeFilter);
+    };
+
+    const onChange = (e: React.ChangeEvent<HTMLInputElement> | React.KeyboardEvent<HTMLSpanElement>) => {
+        const newSelectedOption = (e.target as EventTarget & HTMLInputElement).id;
+
+        if (changedOptions.includes(newSelectedOption)) {
+            setChangedOptions([...changedOptions.filter(item => item !== newSelectedOption)]);
+        } else {
+            setChangedOptions([...changedOptions, newSelectedOption]);
+        }
+    };
+
+    const chipLabelFormatting = () => {
```

Comment:
```suggestion
    const getChipLabel = () => {
```

Reply:
Will rename thank you) I'm always bad at naming))))

Reply:
Done